azure-pipeline-templates/build-release.yml

```diff
@@ -44,6 +44,22 @@ steps:
     displayName: "Test Binary built"
     continueOnError: false
 
+  # Build the health-monitor binary  
+  - task: Go@0
+    inputs:
+      command: 'build'
+      arguments: "-o healthmon ./tools/health-monitor/"
+      workingDirectory: ${{ parameters.work_dir }}
+    displayName: 'Building health-monitor binary'
+  
+  # Verifying whether built binary is correct
+  - script: |
+      sudo chmod +x ./healthmon
```

Comment:
why `healthmon` instead of just `healthmoniter`

Reply:
I kept the name of the binary as healthmon for now. In one of the above comments, @vibhansa-msft suggested to come up with some better name. So, we can think of what name should we have for the binary